src/documents/ConnectionPolicy.ts

```diff
@@ -1,36 +1,37 @@
-import { RetryOptions } from "../retry/retryOptions";
+import { defaultRetryOptions, RetryOptions } from "../retry/retryOptions";
 import { ConnectionMode } from "./ConnectionMode";
-import { MediaReadMode } from "./MediaReadMode";
 /**
  * Represents the Connection policy associated with a CosmosClient in the Azure Cosmos DB database service.
  */
-export class ConnectionPolicy {
-  private static readonly defaultRequestTimeout: number = 60000;
-  private static readonly defaultMediaRequestTimeout: number = 300000;
-
+export interface ConnectionPolicy {
   /** Determines which mode to connect to Cosmos with. (Currently only supports Gateway option) */
-  public ConnectionMode = ConnectionMode.Gateway;
-  /** Attachment content (aka media) download mode. Should be one of the values of {@link MediaReadMode} */
-  public MediaReadMode: keyof typeof MediaReadMode = MediaReadMode.Buffered;
-
-  /** Time to wait for response from network peer for attachment content (aka media) operations. Represented in milliseconds. */
-  public MediaRequestTimeout = ConnectionPolicy.defaultMediaRequestTimeout;
+  ConnectionMode?: ConnectionMode;
```

Comment:
@christopheranderson How do you feel about making all CosmosClientOptions snakeCase? This would be a good time to do it

Reply:
Resolved in person. We are switching all options to use snakeCase